charts/hail-search/values.yaml

```diff
@@ -54,3 +54,8 @@ tolerations: []
 
 global:
   seqrPlatformDeploy: false
+
+exports:
+  environment:
+    global:
```

Comment:
We also might be able to just export this as as `hail-search` value:

exports:
  hail-search:
    environment:
      ...
```